packages/opentelemetry-exporter-collector/README.md

```diff
@@ -49,6 +49,32 @@ provider.register();
 
 ```
 
+By default, plaintext connection is used. In order to use TLS, provide `credentials` option like so:
```

Comment:
Please add a note similar to "to see how to generate credentials, you can refer to the script used to generate certificates for tests [here](link to certs sh)."